### PR TITLE
Fix TO Go DS SSL key to also create latest version

### LINKED ISSUE #2828: TO Golang does not set version to "latest" when generating a cert, rendering the endpoint useless
When using the TO API to generate a certificate for a delivery service via Traffic Portal, the TO Go API does not add the new cert into Riak under the `ssl/<xml_id>-latest` key. Instead, it's getting added under the key `ssl/<xml_id>-1`. So for all intents and purposes this new cert does not exist to TO at all, because everything in TO checks for the `-latest` version of the cert.

This is a regression from the Perl version of `/api/*/deliveryservices/sslkeys/generate` which would put the generated cert in Riak under the requested version (e.g. `ssl/<xml_id>-<requested version>`) as well as the `-latest` version (e.g. `ssl/<xml_id>-latest`).

### PR DESCRIPTION
**Pull request by rob05c**
Fixes #2828

#### What does this PR do?

Fixes #2828

#### Which TC components are affected by this PR?

- [ ] Documentation
- [ ] Grove
- [ ] Traffic Analytics
- [ ] Traffic Monitor
- [x] Traffic Ops
- [ ] Traffic Ops ORT
- [ ] Traffic Portal
- [ ] Traffic Router
- [ ] Traffic Stats
- [ ] Traffic Vault
- [ ] Other _________

#### What is the best way to verify this PR?

Verify #2828

#### Check all that apply

- [ ] This PR includes tests
- [ ] This PR includes documentation updates
- [ ] This PR includes an update to CHANGELOG.md
- [ ] This PR includes all required license headers
- [ ] This PR includes a database migration (ensure that migration sequence is correct)
- [ ] This PR fixes a serious security flaw. Read more: [www.apache.org/security](http://www.apache.org/security/)

### CODE REVIEW

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/trafficcontrol-PR/2452/
Test PASSed.

File: traffic_ops/traffic_ops_golang/deliveryservice/sslkeys.go [hunk 1]
```diff
@@ -51,6 +51,8 @@ func GenerateSSLKeys(w http.ResponseWriter, r *http.Request) {
 	api.WriteResp(w, r, "Successfully created ssl keys for "+*req.DeliveryService)
 }
 
+const DSSSLKeyVersionLatest = `latest`
```
[the remark this answers is not in the document]
Comment (rob05c) [reply traffic_ops/traffic_ops_golang/deliveryservice/sslkeys.go]: Eh, I saw that, but "default" isn't quite the same as "latest". Though I don't think it'd be confusing for the existing usage to set "Latest" if it doesn't exist. How about renaming that to `riaksvc.LatestDSSSLKeyVersion`?

[the remark this answers is not in the document]
Comment (rawlinp) [reply traffic_ops/traffic_ops_golang/deliveryservice/sslkeys.go]: Yeah, that sounds good to me

[the remark this answers is not in the document]
Comment (rob05c) [reply traffic_ops/traffic_ops_golang/deliveryservice/sslkeys.go]: Eh, I saw that, but "default" isn't quite the same as "latest". How about `riaksvc/dsutil.go`
```
const DSSSLKeyVersionLatest = "latest"
const DefaultDSSSLKeyVersion = DSSSLKeyVersionLatest
```
?

[the remark this answers is not in the document]
Comment (rob05c) [reply traffic_ops/traffic_ops_golang/deliveryservice/sslkeys.go]: Done

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/trafficcontrol-PR/2453/
Test PASSed.

**Pull request merged**